server-interfaces/src/ausPost.d.ts

```diff
@@ -0,0 +1,42 @@
+declare namespace server {
```

Comment:
I dont think you need this here. These interfaces describe how the app should interface with the api. Will the app be sending data to Aus Post directly?